src/CaptureClient/CaptureEventProcessor.cpp

```diff
@@ -147,14 +147,8 @@ void CaptureEventProcessorForListener::ProcessEvent(const ClientCaptureEvent& ev
     case ClientCaptureEvent::kModuleUpdateEvent:
       ProcessModuleUpdate(event.module_update_event());
       break;
-    case ClientCaptureEvent::kCgroupMemoryUsage:
-      // TODO (http://b/185107371): Process the cgroup memory usage information.
-      break;
-    case ClientCaptureEvent::kProcessMemoryUsage:
-      // TODO (http://b/185107593): Process the process memory usage information.
-      break;
-    case ClientCaptureEvent::kSystemMemoryUsage:
-      ProcessSystemMemoryUsage(event.system_memory_usage());
```

Comment:
I kind of think we should probably add an explicit `default: break;` for this break, so that we explicitly do not process anything that was removed or added in capture file.

Reply:
Good point! Added accordingly. Thanks!